generatebundlefile/data/bundles_staging/1-25.yaml

```diff
@@ -72,14 +72,14 @@ packages:
         repository: cluster-autoscaler/charts/cluster-autoscaler
         registry: public.ecr.aws/w9m0f3l5
         versions:
-            - name: 9.21.0-1.25-f9017a890ac64959308c86f341b72cdc2c3a67a4
+            - name: 9.34.0-1.25-f9017a890ac64959308c86f341b72cdc2c3a67a4
   - org: kubernetes-sigs
     projects:
       - name: metrics-server
         repository: metrics-server/charts/metrics-server
         registry: public.ecr.aws/w9m0f3l5
         versions:
-            - name: 0.6.2-eks-1-25-9-da99883b9791fb9e8a6e135513f49b636583e40a
+            - name: 0.6.4-eks-1-25-9-da99883b9791fb9e8a6e135513f49b636583e40a
```

Comment:
same as above - we need to make sure this tag exists and represents a recent build.